### DISCUSSION TITLE
React email with Firebase Functions (with Resend)

### DISCUSSION THREAD

**Discussion by MartinXPN** [resend/react-email#741] [Q&A] [answered]
Dear React Email Team,
Thanks a lot for the amazing work you've done. I'm trying to incorporate `react-email` along with `resend` into one of my projects. 

The project uses Firebase Functions as a backend and the emails need to be triggered on Firebase Functions. Yet, when I try to import an email template (a `.tsx` file) into the function code, I get an error:
```shell
⚠  functions: Error: Cannot find module '../../emails/emails/stripe-welcome'
```

My editor is able to locate the module just fine. Yet, the code doesn't run.

It would be great to have a working reference/example of react-email working with Firebase.
Here is my project structure as a reference:
```
functions
|-> emails (react-email folder includes `emails`, `.react-email`, and `static` as subdirectories)
|-> src (includes the main Firebase Functions logic `index.ts`, etc)
|-> test (jest tests)
| + `package.json`, `tsconfig`, etc
```

Let me know if more context needs to be provided. 
Seems like this Stackoverflow question might be relevant: https://stackoverflow.com/questions/75890200/importing-a-jsx-file-in-a-node-js-firebase-function

**Comment by artanisdesign** [comment]
you need to transpile those .. for example with babel, then you can import them okay..

**Comment by MartinXPN** [comment, marked as answer]
Here is hwo to do it. No need for any tricks: https://github.com/MartinXPN/nextjs-firebase-mui-starter
